src/index.html

```diff
@@ -22,16 +22,11 @@ <h2>Recent scores</h2>
       <form action="#" class="add-scores">
         <h2>Add your score</h2>
         <div class="input">
-          <input id="username" placeholder="Your name" required />
-          <input id="user-score" type="number" placeholder="Your score" required />
-          <input class="submit" id="submit" type="submit" value="Submit" />
-        </div>
+          <input id="username" class="inputs" placeholder="Your name" required />
+          <input id="user-score" class="inputs" type="number" placeholder="Your score" required />
+         </div> 
+          <button class="submit" id="submit" type="submit" value="Submit">Submit</button>
```

Comment:
- [ ] Your current form lacks validation and allows users to send values of any length, which breaks the design of the app. Please make sure to add validation and handle long strings by either giving the sections maximum widths or trimming the text. Currently, the app design is broken by a large input.